src/App.js

```diff
@@ -32,6 +33,7 @@ export default function App() {
             <Home />
           </Route>
           <Route component={Book} path="/book/:isbn"></Route>
```

Comment:
I believe it would be cleaner and more visibile if you leave the "path" as the first parameter in route, cause you might need to tweak the "component" later